parity.doc/parity.1

```diff
@@ -0,0 +1,526 @@
+.\" Automatically generated by Pod::Man 4.09 (Pod::Simple 3.35)
```

Comment:
Why commit the generated file? Should'nt the create_man_page.sh be called during the build?